.vsts-ci/linux.yml

```diff
@@ -80,24 +80,17 @@ jobs:
   - template: ../tools/releaseBuild/azureDevOps/templates/insert-nuget-config-azfeed.yml
 
   - powershell: |
-      tools/travis.ps1 -Stage Bootstrap
+      Import-Module ./tools/ci.psm1
```

Comment:
Interesting, auto-loading does not work?

Reply:
Because it is now a PSM1

Reply:
@iSazonov 	It's not in the module path.